test/fixtures/forms/high-form-views-low-conversions.js

```diff
@@ -335,6 +335,63 @@ const testData = {
       ],
     },
   },
+  lowFormviewsAuditData: {
+    type: 'high-page-views-low-form-nav',
+    siteId: 'site-id',
+    auditId: 'audit-id',
+    auditResult: {
+      formVitals: [
+        {
+          url: 'https://www.surest.com/high-page-low-form-view',
+          formsubmit: {
+            'desktop:windows': 0,
+          },
+          formview: {
+            'desktop:windows': 200,
+          },
+          formengagement: {
+            'desktop:windows': 100,
+          },
+          pageview: {
+            'desktop:windows': 5690,
+            'mobile:ios': 1000,
+          },
+          trafficacquisition: {
+            paid: 2690,
+            maxTimeDelta: 3060,
```

Comment:
maxTimeDelta?

Reply:
This is audit test data, usually the trafficacquisition object has maxTimeDelta and sources apart from the ones we need for traffic metric. Later we only use the paid, earned, total and owned

Reply:
Got that. Let's remove these extra props from from form vitals itself jna subsequent pr.